app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def show_errors(object, field_name)
+    if object.errors.any?
+      if !object.errors.messages[field_name].blank?
```

Comment:
Combine the nested conditional here with an `&&` clause.  Also, slight preference for using `present?` over `!blank?`.